src/ape/utils/misc.py

```diff
@@ -355,6 +355,7 @@ def raises_not_implemented(fn):
     def inner(*args, **kwargs):
         raise _create_raises_not_implemented_error(fn)
 
+    inner.__doc__ = fn.__doc__
```

Comment:
it may be worth putting a comment here saying this is needed for the doc strings to show up